src/views/List.jsx

```diff
@@ -32,9 +32,16 @@ export function List({ data }) {
 					X
 				</button>
 			)}
+			<p>Mark your purchases, they will automatically uncheck after 24hrs.</p>
+			<p>You can uncheck if you no longer want to make the purchase.</p>
```

Comment:
This instruction is unclear to me as a user. What exactly are you saying? From my point of view I added the item to the list because I wanted to purchase it, so I figure I will check it once it has been purchased. This could be causing some confusion on my understanding of the functionality of the app.

Reply:
This was an attempt to explain that they don't need to uncheck after they purchase the item, that the app will do that for them.

But if they accidentally check the item, then they should uncheck it themselves.

Reply:
This clarification helps, but I am going to think more on how I would word this and I'll come back. I think also my understanding of the functionality impacted my understanding of the instructions.

Reply:
That'd be great! I wonder if this would be clearer? 

"Check off items as you shop, your list will reset after 24hrs"
"Only manually uncheck if you didn't make the purchase"